react/components/Toast/index.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Toast from './Toast'
+import isString from 'lodash/isString'
+
+const ToastContext = React.createContext({
+  showToast: () => {},
+  hideToast: () => {},
+})
+
+class ToastProvider extends Component {
+  state = {
+    toast: null,
+    nextToast: null,
+    toastVisible: true,
+  }
+
+  showToast = (args) => {
+    if (isString(args)) {
+      args = { message: args }
+    }
+    const { message = '', action } = args
+
+    // There was a duration option, that has been disabled for now.
+    // If there is ever a need to turn it back on, you can include it
+    // back like this:
+    // const { message = '', action, duration } = args
+
+    if (this.state.toast) {
+      // If there is a toast present already, queue up the next toast
+      // It will be displayed when the current toast is closed, on handleToastClose
+      this.setState({
+        nextToast: {
+          message,
+          action,
+          // duration,
```

Comment:
Não seria melhor remover essa linha de código comentada?

Reply:
Concordo com @vitoriaHeliane , @lbebber  poderia retirar o comentário dentro da mudança de estado e explicar fora dele, como nos comentários acima, onde colocar o `duration` se for necessário.

Reply:
acho que vou remover todo esse comentario sobre o duration mesmo, nao sei se vai ter necessidade disso, e se tiver, não é dificil implementar